peerberrypy/api.py

```diff
@@ -518,11 +518,11 @@ def get_account_summary(
     def get_transactions(
             self,
             quantity: int,
-            start_page: int,
-            start_date: date,
-            end_date: date,
-            periodicity: str = None,
-            transaction_types: list = None,
+            start_page: int = 0,
+            start_date: Optional[date] = None,
+            end_date: Optional[date] = None,
+            periodicity: Optional[str] = None,
+            transaction_types: Optional[list[str]] = None,
```

Comment:
Instead of having `list[str]`, it's better to have `List[str]` to ensure backward compatibility with older Python versions. You can use the List from the typing module.